config/env.js

```
@@ -18,4 +21,17 @@ module.exports = function (express, app) {
   app.use(bodyParser.json());
   app.use(methodOverride());
   app.use(express.static(path.join(__dirname, 'public')));
+
+  app.set('restricted', restricted);
+  if (restricted) {
+    var passport = require('passport')
+      , BasicStrategy = require('passport-http').BasicStrategy;
+    passport.use(new BasicStrategy(function(username, password, done) {
+      if (username !== restrictedUsername || password !== restrictedPassword) {
+        return done(null, false);
```

with this being the failure case, do you want to throw an actual error for the first argument instead of passing in null? see the type definition for the response: https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/passport-http/index.d.ts